config.yaml

```
@@ -0,0 +1,9 @@
+author: Deepak Sharma <deepshar@redhat.com>
+server: "https://f8a-analytics-preview-2445582058137.production.gw.apicast.io:443"
```

should we call this as host instead of server?
```suggestion
host: "https://f8a-analytics-preview-2445582058137.production.gw.apicast.io:443"
```

---

ack.